delta-lake/common/src/main/delta-33x-40x/scala/com/nvidia/spark/rapids/delta/common/RapidsRowIndexFiltersBase.scala

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
Unless we are planning on actually extending these Filters in Delta 4, I would just move them here and keep the same name, the files can directly be referenced in the Gpu version of DeltaParquetFileFormat

Reply:
Done